features/step_definitions/back_office/edit_steps.rb

```diff
@@ -0,0 +1,74 @@
+require "pry"
+
+When("I edit the most recent registration") do
+  @edited_info = {
+    contact_email: generate_email
+  }
+
+  @bo.dashboard_page.view_reg_details(search_term: @reg_number)
+
+  @bo.registration_details_page.edit_link.click
+  @bo.edit_page.change_contact_email.click
+
+  @journey.contact_email_page.submit(email: @edited_info[:contact_email], confirm_email: @edited_info[:contact_email])
+end
+
+When("I confirm the changes") do
+  @bo.edit_page.confirm_changes.click
+  @journey.declaration_page.submit
+end
+
+Then("I can see the changes on the registration details page") do
+  find_link("View registration").click
+
+  @edited_info.each_value do |value|
+    expect(page).to have_text(value)
+  end
+end
+
+When("I edit the most recent registration type to {string}") do |carrier_type|
+  edited_text = case carrier_type
+                when "carrier_dealer"
+                  "Carrier and dealer"
+                when "broker_dealer"
+                  "Broker and dealer"
+                else
+                  "Carrier, broker and dealer"
+                end
+
+  @edited_info = {
+    carrier_type: edited_text
+  }
+
+  @bo.dashboard_page.view_reg_details(search_term: @reg_number)
+
+  @bo.registration_details_page.edit_link.click
+  @bo.edit_page.change_registration_type.click
+
+  @journey.carrier_type_page.submit(choice: carrier_type)
+end
+
+When("I pay by card") do
```

Comment:
These steps are specific to the edit journey, although they look like they could be reused elsewhere. So I'd suggest:

- If they're general payment steps then move them to finance_payment_steps
- If they're only to be used for editing then rename them, e.g. `I pay for the edit by card / bank transfer`

Reply:
The actual implementation 
```
When("I pay by card") do
  @bo.payment_summary_page.submit(choice: :card_payment)

  submit_valid_card_payment
end
```
only uses pages and methods already shared, and so this is applicable elsewhere where there is a payment summary page. Hence this is not very specific to edits, can be used in any registration journey and even for the copy cards. Happy to rename if you think is better, but to me, this looks pretty generic and reusable :)

Reply:
Let me know and I'll open a different PR to rename it :) Merging now